### ISSUE TITLE
7. Accounting for Context - [E3] 3. M. Liu, Y. Bu, C. Chen, J. Xu, D. Li, Y. Leng, R. Freeman...Y. Ding. 2020.

### ISSUE THREAD

**Issue by JunsolKim** [UChicago-Computational-Content-Analysis/Readings-Responses-2023#18]
Post questions here for this week's exemplary readings: 3. M. Liu, Y. Bu, C. Chen, J. Xu, D. Li, Y. Leng, R. Freeman...Y. Ding. 2020. “Can Pandemics Transform Scientific Novelty? Evidence from COVID-19.” arXiv:2009.12500.

**Comment by konratp** [comment]
This was an interesting read, though I'm beginning to really get lost in the methodologies used in these papers (even though I'm well-aware that that is the reason these paper's are assigned). In my very limited understanding of the matter though, it was not clear to me why the researchers would choose countries as their levels of analyses. Especially during the COVID-19 pandemic, it became evident that it was collaboration across national boundaries that could help humanity out of this crisis. Moreover, academia in general is a highly international subset of society, with researchers often working in countries whose citizenship they do not hold. Would it not be better to choose other levels of analysis (e.g. individual institutions) rather than countries?

**Comment by Sirius2713** [comment]
> This was an interesting read, though I'm beginning to really get lost in the methodologies used in these papers (even though I'm well-aware that that is the reason these paper's are assigned). In my very limited understanding of the matter though, it was not clear to me why the researchers would choose countries as their levels of analyses. Especially during the COVID-19 pandemic, it became evident that it was collaboration across national boundaries that could help humanity out of this crisis. Moreover, academia in general is a highly international subset of society, with researchers often working in countries whose citizenship they do not hold. Would it not be better to choose other levels of analysis (e.g. individual institutions) rather than countries?

I agree with you that it's common for researchers to collaborate globally, even before the pandemic. But I think this paper uses country-level data because researchers generally tended to work domestically due to the hassle of traveling, visa etc. And many online collaboration tools like Zoom emerges during the pandemic, which makes it easier and much more convenient to collaborate globally. Instead of institutional level analysis, I'm more interested in the collaboration outside established circles like colleagues, mentorship etc.

**Comment by Jiayu-Kang** [comment]
I'm wondering how we should interpret the findings of this study, other than knowing that science progresses differently before vs. during the pandemic. The discussion of the results seems a bit unclear to me in terms of what might explain the change in the relationship between collaboration and novelty.

**Comment by facundosuenzo** [comment]
I'm curious to hear your thoughts on how novelty was measured: 
"novelty score indicates the proportion of entity pairs that are highly distant to the possible entity pairs in a paper."
Is it a robust way of conceptualizing it? What could other ways be?

**Comment by mikepackard415** [comment]
I have some non-technical thoughts on this paper. As usual I find myself thinking about climate change and other environmental crises. If the immediate and pressing threat of covid seems to have sped up scientific innovation, how might this map onto more slow-moving (but more threatening, in my opinion) situations? It's important to remember that we also want to avoid burnout for scientists, so I think more science is not always the answer. Can we get more novelty without necessarily just increasing our pace?

**Comment by Hongkai040** [comment]
Well maybe the authors are right. But I don’t like the title…Maybe the pandemic forced us to work together to come up some solutions to fight against the pandemic.  I don’t think it will be necessarily the case in other fields. 
And I don’t understand why the difference between different bio-entities could be the novelty score of a paper? Can anybody tell me? Thanks!

**Comment by NaiyuJ** [comment]
Okay, it is true that Covid facilitates scientific collaboration and the inquiry of distant resources. But I feel like Covid itself has given a lot of interesting topics that scholars can think about in the fields of economics, public health, sociology, and political science. There are a lot of scholars starting to study Covid and public health. This is hard to measure.

**Comment by sizhenf** [comment]
I wonder how scientific novelty in other fields have changed over Covid-19, since it seems like the paper only looked at Coronavirus-related papers.

**Comment by YileC928** [comment]
I feel that the paper does not clearly interpret their results and it does not provide plausible explanations regarding mechanisms of why novelties that are associated with first-time/international collaboration tend to increase during the pandemic.

**Comment by kelseywu99** [comment]
Second what Hongkai has said above, there's a bit of reverse causation going on in the title... That being said, I find their research methods interesting but the interpretation of results rather opaque. Would like to see further elaboration on this.

**Comment by LuZhang0128** [comment]
I feel like Coivd itself opens a lot of discussion in multiple fields, as well as its consequences. Like the current war between Russia and Ukraine, I can foresee scholars from history, political science, economics, etc. publishing papers on this topic. I wonder how to measure its **real** contribution to the academic field.

**Comment by chentian418** [comment]
I was wondering whether the parallel trend assumption is satisfied in the model, i.e., if the Covid outbreak hadn't happen, will the novelty scores of the treatment and control groups remain a constant difference?

**Comment by ttsujikawa** [comment]
I am not sure if a change in first-time/international collaborations could be major reason of higher novelties of paper. There should be more dominant reasons why novelties of paper have increased during the pandemic. Due to it uniqueness and immense impacts on society, researchers and government were forced to be active in social science research. Such that, there should be a lot of more hypothesis on dominant reason for the consequences.

**Comment by melody1126** [comment]
The novelty score that the authors used seem to be central to the results. It seems like a regular regression equation with interaction terms (page 11). Here, why is "time collaboration" a negative term? Why is team size also taken into account? 

𝑁𝑜𝑣𝑒𝑙𝑡𝑦 𝑠𝑐𝑜𝑟𝑒𝑖 = 𝛼 + 𝛽1 𝐶𝑂𝑉𝐼𝐷19 + 𝛽2 𝑓𝑖𝑟𝑠𝑡 − 𝑡𝑖𝑚𝑒 𝑐𝑜𝑙𝑙𝑎𝑏𝑜𝑟𝑎𝑡𝑖𝑜𝑛 +
𝛽 𝑖𝑛𝑡𝑒𝑟𝑛𝑎𝑡𝑖𝑜𝑛𝑎𝑙 𝑐𝑜𝑙𝑙𝑎𝑏𝑜𝑟𝑎𝑡𝑖𝑜𝑛 + 𝛽 𝑓𝑖𝑟𝑠𝑡_𝐶 × 𝐶𝑂𝑉𝐼𝐷19 + 𝛽 𝑖𝑛𝑡𝑒𝑟𝑛𝑎𝑡𝑖𝑜𝑛𝑎𝑙_𝐶 × 345
𝐶𝑂𝑉𝐼𝐷19 + 𝛽6𝑡𝑒𝑎𝑚 𝑠𝑖𝑧𝑒 + 𝛿𝑡 + 𝜀 (5)

**Comment by chuqingzhao** [comment]
I have the similar concern of the novelty measurement in this paper. The cosine similarity measure the semantic similarity/dissimilarity between two papers. I am not sure what is the baseline of novelty comparison? On the other hand, what the measurement seems to do is to capture the dissimilar papers, but I also wonder dissimilarity could be equal to novelty?

**Comment by chuqingzhao** [comment]
> The novelty score that the authors used seem to be central to the results. It seems like a regular regression equation with interaction terms (page 11). Here, why is "time collaboration" a negative term? Why is team size also taken into account?
> 
> 𝑁𝑜𝑣𝑒𝑙𝑡𝑦 𝑠𝑐𝑜𝑟𝑒𝑖 = 𝛼 + 𝛽1 𝐶𝑂𝑉𝐼𝐷19 + 𝛽2 𝑓𝑖𝑟𝑠𝑡 − 𝑡𝑖𝑚𝑒 𝑐𝑜𝑙𝑙𝑎𝑏𝑜𝑟𝑎𝑡𝑖𝑜𝑛 + 𝛽 𝑖𝑛𝑡𝑒𝑟𝑛𝑎𝑡𝑖𝑜𝑛𝑎𝑙 𝑐𝑜𝑙𝑙𝑎𝑏𝑜𝑟𝑎𝑡𝑖𝑜𝑛 + 𝛽 𝑓𝑖𝑟𝑠𝑡_𝐶 × 𝐶𝑂𝑉𝐼𝐷19 + 𝛽 𝑖𝑛𝑡𝑒𝑟𝑛𝑎𝑡𝑖𝑜𝑛𝑎𝑙_𝐶 × 345 𝐶𝑂𝑉𝐼𝐷19 + 𝛽6𝑡𝑒𝑎𝑚 𝑠𝑖𝑧𝑒 + 𝛿𝑡 + 𝜀 (5)

It's a fair concern. From my understanding, the team size is a control here--the notion is that if large team could develop more innovative paper because they could incorporate more labors or knowledge than small teams. "first - time collaboration" means the first-time collaboration ration for a paper; if larger the score is, the more  first-time collaboration involved in the team.
